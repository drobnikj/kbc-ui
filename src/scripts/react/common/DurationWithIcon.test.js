import React from 'react';

import DurationWithIcon from './DurationWithIcon';

describe('<DurationWithIcon />', function() {
  it('should render with no props', function() {
    shallowSnapshot(
      <DurationWithIcon />
    );
  });

  it('should render with start prop', function() {
    shallowSnapshot(
      <DurationWithIcon
        startTime="2017-09-23T12:30:00+00:00"
      />
    );
  });

  it('should render with end props', function() {
    shallowSnapshot(
      <DurationWithIcon
        endTime="2017-09-23T14:30:00+00:00"
      />
    );
  });

  it('should render with both props', function() {
    shallowSnapshot(
      <DurationWithIcon
        startTime="2017-09-23T12:30:00+00:00"
        endTime="2017-09-23T14:30:00+00:00"
      />
    );
  });
});
