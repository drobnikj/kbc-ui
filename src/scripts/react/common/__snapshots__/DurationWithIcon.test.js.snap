exports[`<DurationWithIcon /> should render with both props 1`] = `
<span>
  <i
    className="fa fa-clock-o" />
   
  <Duration
    endTime="2017-09-23T14:30:00+00:00"
    startTime="2017-09-23T12:30:00+00:00" />
</span>
`;

exports[`<DurationWithIcon /> should render with end props 1`] = `
<span>
  <i
    className="fa fa-clock-o" />
   
  <Duration
    endTime="2017-09-23T14:30:00+00:00" />
</span>
`;

exports[`<DurationWithIcon /> should render with no props 1`] = `
<span>
  <i
    className="fa fa-clock-o" />
   
  <Duration />
</span>
`;

exports[`<DurationWithIcon /> should render with start prop 1`] = `
<span>
  <i
    className="fa fa-clock-o" />
   
  <Duration
    startTime="2017-09-23T12:30:00+00:00" />
</span>
`;
